db/seeds/development.rb

```diff
@@ -27,6 +27,12 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
+# Disable mail delivery for the duration of this task
+ActionMailer::Base.perform_deliveries = false
+
+# Avoid asynchronous DeliverWorkPackageCreatedJob
+Delayed::Worker.delay_jobs = false
```

Comment:
This doesn't simply disable delayed jobs, it force them to run inline on the same process. This is essentially what's used for testing.